layers/+tools/shell/funcs.el

```diff
@@ -209,6 +209,10 @@ is achieved by adding the relevant text properties."
   (switch-to-buffer "*shell*")
   (shell "*shell*"))
 
+(defun spacemacs/open-vterm (&optional ARG)
```

Comment:
`spacemacs/open-vterm` might be unnecessary - we may be able to use `vterm` function directly.

Reply:
Tried again, it worked, it's fixed ✨